### PR TITLE
Pulled error handling from flarum/core

### PR DESCRIPTION
**Pull request by kirkbushell**
Added first test for exception handler
Added status code check for response bag
Added status code check for response bag
Tests for response without debug mode
Added test for debug mode

### CODE REVIEW

File: src/Exception/Handler/FallbackExceptionHandler.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Tobscure\JsonApi\Exception\Handler;
+
+use Exception;
+use Flarum\Core;
```
Comment (franzliedke) [inline src/Exception/Handler/FallbackExceptionHandler.php]: Whoopsie. ;)

**Comment by franzliedke** [comment]
Looks good otherwise. Thanks! :+1:

**Comment by kirkbushell** [comment]
And up goes the commit log again. To hell rebasing this one again.

**Comment by tobscure** [comment]
Thanks Kirk! Sorry for putting you through the squash pain... this is fine, I'll fix the CS issues.

**Pull request merged**